Base/usr/share/man/man5/drag-and-drop.md

```diff
@@ -0,0 +1 @@
+clipboard.md
```

Comment:
What's this?

Reply:
... supposed to be a symlink, git might have messed that up during cherry-pick

Reply:
It IS a symlink according to my local checkout and git, so I think that's just GitHub.